content/creating-user-centred-content/understanding-diversity/why-be-inclusive.md

```diff
@@ -15,23 +15,21 @@ Diverse groups include:
 - people from culturally and linguistically diverse (CALD)
 backgrounds
 - people with cognitive or physical disability
-- People who are blind or have a vision impairment 
-- People who are deaf or have a hearing impairment
+- people who are blind or have a vision impairment 
+- people who are deaf or have a hearing impairment
 - people living in rural or remote locations
 - people accessing the internet on mobile devices
 
 ### [2]Accessibility
 
-Accessibility issues affect many Australians across a variety of demographics.
+Accessibility issues affect many users across a variety of demographics.
 
 Accessibility issues may include:
 - web content that is difficult to understand
 - websites not tailored to screen readers
 - websites that are difficult to navigate using smartphones
 - content too big to download with a low data limit
 
-By addressing accessibility and diversity, we can make our content simple and easy for everyone to understand. 
+By addressing accessibility and diversity, we can make our content simple and easy to understand. 
 
 The [Content Guide](https://guides.service.gov.au/content-guide/accessibility-inclusivity/#audience-diversity) has guidance about creating accessible and inclusive content.
```

Comment:
Can we make this better link text for screen readers and SEO? (to reflect that the link is going to the `accessibility and inclusivity` section, not to the top level Content Guide?